recipes/pangolin/build.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+
+mkdir build
+cd build
+
+cmake .. \
+    -G "Ninja" \
+    -DCMAKE_BUILD_TYPE=Release \
+    -DCMAKE_INSTALL_LIBDIR="lib" \
+    -DCMAKE_INSTALL_PREFIX=$PREFIX \
+    -DCMAKE_PREFIX_PATH=$PREFIX \
+    -DBUILD_EXAMPLES=OFF \
+    -DBUILD_TESTS=OFF \
+    -DBUILD_TOOLS=OFF
+
+cmake --build . --config Release
```

Comment:
Is there a `make check` or `ctest` here we can run?

Reply:
The upstream library has tests, but they are not configured to build and run. Would it be acceptable to create a feedstock without these tests and add them later for faster iteration?

Reply:
If there are tests, ideally we should have them here first. Is that something you can tackle?

Reply:
Yes, I will add the tests to this diff. Thank you for your guidance

Reply:
Done!

https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=1029198&view=logs&j=6f142865-96c3-535c-b7ea-873d86b887bd&t=22b0682d-ab9e-55d7-9c79-49f3c3ba4823&l=1195